src/emulator/modules/SceTouch/SceTouch.cpp

```diff
@@ -24,14 +24,95 @@
 
 // TODO Move elsewhere.
 static uint64_t timestamp;
+SDL_TouchFingerEvent finger_buffer[8];
+uint8_t finger_count = 0;
+long int finger_id_ref;
+auto touchscreen_port = SCE_TOUCH_PORT_FRONT;
+
+SceTouchData recover_touch_events() {
+    SceTouchData touch_data;
+    touch_data.timeStamp = timestamp;
+
+    for (int i = 0; i < finger_count; i++) {
```

Comment:
This can create a signed/unsigned comparison warning.

```suggestion
    for (uint8_t i = 0; i < finger_count; i++) {
```